debian/ceph-common.postinst

```diff
@@ -62,6 +62,8 @@ case "$1" in
                -d $SERVER_HOME   \
                -g $SERVER_GROUP  \
                $SERVER_USER
+       # Unlock $SERVER_USER in case it is locked from an uninstall
+       usermod -e 99999 -U $SERVER_USER
```

Comment:
please drop the `-e 99999` option. and instead conditionalize this by `/etc/shadow`, like
```sh
    if [ -f /etc/shadow ]; then 
      usermod -U -e '' $SERVER_USER 
    else 
      usermod -U $SERVER_USER
    fi 
```